Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -176,6 +175,39 @@ final class ExperimentalAnimationLayer: BaseAnimationLayer {
       context: layerContext)
   }
 
+  /// Immediately builds and begins playing `CAAnimation`s for each sublayer
+  private func setupAnimation(for configuration: AnimationConfiguration) {
```

Comment:
This is now only called a maximum of once per run loop cycle, from `display()`

Reply:
Nice!